eoc_journal/eoc_journal.py

```diff
@@ -30,17 +31,14 @@
 from .completion_api import CompletionApiClient
 from .course_blocks_api import CourseBlocksApiClient
 from .pdf_generator import get_style_sheet
-from .utils import _, normalize_id
+from .utils import _, normalize_id, DummyTranslationService
 
 try:
     from django.contrib.auth.models import User
 except ImportError:
     User = None  # pylint: disable=C0103
 
 
-loader = ResourceLoader(__name__)
-
-
```

Comment:
What is the reason for this change? Please undo it since it creates an unnecessary diff.